### PR TITLE
fix: OpenJPH compression issues - temporarily publish debug

### PR DESCRIPTION
**Pull request by wayfarer3130**
Updated to the latest openjph version, and switched the dicom-codec libraries to use >= includes for easy linking of test versions.  Seems to improve the HTJ2K stability enormously.

### CODE REVIEW

File: packages/dicom-codec/package.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
     "@cornerstonejs/codec-charls": ">=1.2.3",
     "@cornerstonejs/codec-libjpeg-turbo-8bit": ">=1.2.2",
     "@cornerstonejs/codec-little-endian": ">=0.0.6",
-    "@cornerstonejs/codec-openjpeg": "^1.2.4",
-    "@cornerstonejs/codec-openjph": "^2.4.5",
+    "@cornerstonejs/codec-openjpeg": ">=1.2.4",
```
Comment (wayfarer3130) [inline packages/dicom-codec/package.json:30]: This makes it easy to test newer versions as the linking doesn't get messed up.

**Comment by wayfarer3130** [comment]
Please merge https://github.com/aous72/OpenJPH/pull/149 after this PR is merged/status checked

File: packages/openjphjs/build.sh [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 mkdir -p build
 mkdir -p dist
-#(cd build && CXXFLAGS=-msimd128 emcmake cmake -DCMAKE_BUILD_TYPE=Debug ..)
-(cd build && CXXFLAGS=-msimd128 emcmake cmake ..)
+(cd build && CXXFLAGS=-msimd128 emcmake cmake -DCMAKE_BUILD_TYPE=Debug ..)
```
Comment (wayfarer3130) [inline packages/openjphjs/build.sh:4]: This is temporary until we figure out why the build is failing

File: packages/openjphjs/src/HTJ2KDecoder.hpp [hunk 3]
```diff
@@ -38,7 +38,7 @@ class HTJ2KDecoder
   {
     // Use the following for debugging to ensure updated version info
     // Update the XX to check that reload has completed
-    // OJPH_INFO(0x00010002, "vXX HTJ2K Decoder");
+    OJPH_INFO(0x00010002, "v06 HTJ2K Decoder");
```
Comment (wayfarer3130) [inline packages/openjphjs/src/HTJ2KDecoder.hpp:41]: Temporary while we figure out why the code fails to decode

Comment (sedghi) [reply packages/openjphjs/src/HTJ2KDecoder.hpp:41]: as talked, this makes it bigger but gives stack trace etc

**Review by sedghi — APPROVED** [review]

**Pull request merged**